{
  "contest": 182,
  "handle": "Bnke0x0",
  "risk": "2",
  "title": "Unbounded loops may cause withdraw() to fail",
  "issueId": 232,
  "issueUrl": "https://github.com/code-423n4/2022-11-stakehouse-findings/issues/232"
}